src/test/ui/generic-associated-types/bugs/issue-87735.rs

```diff
@@ -1,5 +1,5 @@
 // check-fail
-// known-bug
+// known-bug: #87735, #88526
```

Comment:
Hm, I didn't notice this before. This is outside the "intended" syntax we're validating; if we want to accept it, I think we should do so intentionally -- and probably update our parser to validate the prefix before # as well (no whitespace?).

Reply:
I've updated the parser. I wasn't able to compiler the tests, but it seems to work as expected in the rust playground.